packages/shared/src/components/modals/PostToSquadModal.tsx

```diff
@@ -51,6 +55,11 @@ function PostToSquadModal({
     preview,
   });
   const { completeAction } = useActions();
+  const isMobile = !useMedia([tablet.replace('@media ', '')], [true], false);
+  let titleText = null;
+  if (!isMobile) {
+    titleText = shouldSkipHistory ? 'Post article' : 'Share post';
+  }
```

Comment:
I think we can simplify it from here then add a condition in the `title` property of the `<Modal.Header`. WDYT?

```suggestion
  const titleText = shouldSkipHistory ? 'Post article' : 'Share post';
```

Reply:
The issue is on Mobile the text needs to be undefined or null so it doesn't show on mobile only, so instead of duplicating the header and conditionally rendering it I did this method, I tried to put all this logic in the title prop but got the linting error "Do not nest ternary expressions" so the above was the solution I came, if you have any other suggestions so that on mobile the title isn't defined please do suggest or if you think i should conditionally render the Modal.Header for mobile and desktop please do say.

Reply:
That linting rule is alright, though you should not get that on two separate ternaries. After the changes above we can then update the part where it should be undefined.

The header should then look like below
```
<Modal.Header showCloseButton={!isMobile} title={isMobile ? null : titleText}>
```

Reply:
I like it, will make the change now, thanks for the suggestion :)